src/ether-utils.js

```diff
@@ -0,0 +1,6 @@
+import { isValidAddress } from 'ethereumjs-util';
+
+// Accepts checksummed addresses too
+const EtherAddress = address => isValidAddress(address);
+
+export default EtherAddress;
```

Comment:
Es lint gives this error then 
4:1  error  Prefer default export  import/prefer-default-export
and the commit fails unless we put it in eslint ignore